apps/api/src/city/city.controller.ts

```diff
@@ -12,4 +13,28 @@ export class CityController {
   async getData() {
     return await this.cityService.listCities()
   }
+
+  @Get('list-one/:id')
+  @Public()
+  findOne(@Param('id') id: string) {
+    return this.cityService.listCity(id)
+  }
+
+  @Post('create-city')
```

Comment:
Let's make only get requests `@Public()` but make all others private by removing the `@Public()` annotation